common/source/docs/common-frsky-flightdeck.rst

```diff
@@ -0,0 +1,43 @@
+.. _common-frsky-flightdeck:
```

Comment:
Can you explain the difference compared to other ArduPilot equipment?
e.g., http://ardupilot.org/copter/docs/common-3dr-power-module.html or http://ardupilot.org/copter/docs/common-rangefinder-lidarlite.html

They all share in common a brief description of the purposes and capabilities, demo videos, links to where to purchase, explanation on how to connect the equipment, and setup through Mission Planner...

There are many other equipments described in the wiki that are not an ArduPilot capability per se... Why should this be different?